modules/default.nix

```diff
@@ -0,0 +1,75 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+with lib; let
+  moduleName = "boulette";
+  cfg = config.services.${moduleName};
+in {
+  options.services.${moduleName} = {
+    enable = mkEnableOption "Enable ${moduleName}";
+    sshOnly = mkOption {
+      type = types.bool;
+      default = false;
+      example = true;
+      description = "Boulette confirmation prompt will be triggerd inside ssh session only.";
+    };
+    challengeType = mkOption {
+      type = types.string;
```

Comment:
```
trace: evaluation warning: The type `types.string` is deprecated. See https://github.com/NixOS/nixpkgs/pull/66346 for better alternative types.

```
Nix just yelled at me about this; I will update this when I get a moment to figure out a better typing.

Reply:
Fixed, this; did an enum on array of valid strings.